### PR TITLE
ECON Mode On 0x221

### PR DESCRIPTION
**Pull request by vanillagorillaa**
Defined ECO Mode on

### CODE REVIEW

File: honda_civic_touring_2016_can.dbc [hunk 1]
```diff
@@ -198,6 +198,8 @@ BO_ 513 GAS_SENSOR: 5 INTERCEPTOR
  SG_ CHECKSUM : 35|4@0+ (1,0) [0|3] "" NEO
 
 BO_ 545 XXX_10: 6 XXX
+ SG_ ECON_ON_2 : 37|2@0+ (1,0) [0|3] "" NEO
```
Comment (rbiasini) [inline honda_civic_touring_2016_can.dbc]: 2 bits? What value corresponds to eco mode on? I would add a value table at the end of dbc to say what 0,1,2,3 mean.

**Comment by vanillagorillaa** [comment]
I haven't seen any other value besides 0=Off and 3=On. Would you like value table added or just remove for now since it already has another message?

**Comment by rbiasini** [comment]
Value table is good, just 3 and 0

**Comment by vanillagorillaa** [comment]
Added value table

**Pull request merged**